lib/honeycomb/integrations/rails.rb

```diff
@@ -89,6 +89,16 @@ class Middleware
       include Rack
       include Warden
       include Rails
+
+      def call_with_hook(env, span, &_add_field)
+        super
+      rescue StandardError => e
+        wrapped = ActionDispatch::ExceptionWrapper.new(nil, e)
+
+        span.add_field "response.status_code", wrapped.status_code
```

Comment:
One final note on this, maybe: there's a *chance* this status code is wrong since a middleware higher in the stack may change it. However, that's always been true given the tracing middleware isn't the first in/last out one.